src/polyswarm/client/scan.py

```diff
@@ -18,7 +17,7 @@ def scan():
 @scan.command('file', short_help='Scan files/directories.')
 @click.option('-r', '--recursive', is_flag=True, default=False, help='Scan directories recursively.')
 @click.option('-t', '--timeout', type=click.INT, default=settings.DEFAULT_SCAN_TIMEOUT,
-              help='How long to wait for results (default: {}).'.format(settings.DEFAULT_SCAN_TIMEOUT))
+              help=f'How long to wait for results (default: {settings.DEFAULT_SCAN_TIMEOUT}).')
```

Comment:
Should we already set `show_envvar=True` here? Or let it be for a future PR ?

Reply:
We have to define what would be the name for that env variable first, here is taking the value from the settings module in the polyswarm-api library, that currently has a hardcoded value of `60*15` (https://github.com/polyswarm/polyswarm-api/blob/9210d15801902eccec64d35338f614761d7a6e89/src/polyswarm_api/settings.py#L9).

Reply:
So yes, should be a different PR I think.